_includes/footer-nolang.html

```diff
@@ -0,0 +1,44 @@
+<footer class="site-footer">
```

Comment:
I think we don't need this. We can just disable the language itself in the dropdown in the footer. To do this, add "id" here: https://github.com/mozilla/activate.mozilla.community/blob/gh-pages/_config.yml#L24 and https://github.com/mozilla/activate.mozilla.community/blob/gh-pages/_config-dev.yml#L24